### PR TITLE
📦 Add package.json for npm

### LINKED ISSUE #20: Add package.json for npm registration
Add a `package.json` file for releasing the IG to npm. Here is an example: https://github.com/anvilproject/client-apis/blob/master/fhir/package.json

### PR DESCRIPTION
**Pull request by bwalsh**
This PR adds a `package.json` file for releasing the IG to npm. This PR also closes #20.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+ "name": "anvil",
```
Comment (liberaliscomputing) [inline package.json:2]: Hehe, could you change this to `ncpi-fhir-ig`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ "version" : "0.1.0",  
+ "canonical" : "https://ncpi-fhir.github.io/ncpi-fhir-ig",
+ "url" : "https://ncpi-fhir.github.io/ncpi-fhir-ig",
+ "title" : "NCPI Implementation Guide",
```
Comment (liberaliscomputing) [inline package.json:6]: NCPI FHIR Implementation Guide

File: package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ "description": "The NIH Cloud-based Platform Interoperability (NCPI) FHIR Working Group Implementation Guide.",
```
Comment (liberaliscomputing) [inline package.json:7]: `"The NIH Cloud Platform Interoperability Effort (NCPI) FHIR Implementation Guide."`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ },
+ "keywords": [
+   "NCPI",
+   "Kids First",   
+   "AnVIL",
+   "GA4GH",
+   "BioInformatics"
+ ],
+ "language": "en",
+ "author": "AnVIL",
```
Comment (liberaliscomputing) [inline package.json:20]: `"NCPI FHIR Working Group"`

File: package.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+ "maintainers": [
+   {
+     "name": "NIH Cloud-based Platform Interoperability (NCPI)",
```
Comment (liberaliscomputing) [inline package.json:23]: `"NIH Cloud Platform Interoperability Effort (NCPI)"`

**Comment by liberaliscomputing** [comment]
@bwalsh, could you add `:package:` in the beginning of your commit message?

**Review by liberaliscomputing — APPROVED** [review]
Looks good!

**Review by torstees — APPROVED** [review]
Looks good to me

**Pull request merged**